app/views/components/add-another/index.html

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-one-half">
+
+      <!-- <h1 class="govuk-heading-l">MOJ Frontend</h1> -->
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "../"
+      }) }}
+
+      <h1 class="govuk-heading-xl" tabindex="-1">
+        <span class="govuk-caption-xl">Components</span> Add another
+      </h1>
+
+      <h2 class="govuk-heading-l moj-add-another__heading">Add a person</h1>
+
+      <form class="moj-add-another-form" method="post" action="#">
+
+        {% call govukFieldset({
+          classes: 'moj-add-another__item',
+          legend: {
+            text: 'Person',
+            classes: 'moj-add-another__title govuk-fieldset__legend--m',
+            isPageHeading: false
+          }
+        }) %}
+
+        {{ govukInput({
+          id: 'person[0][first_name]',
+          name: 'person[0][first_name]',
+          label: {
+            html: 'First name',
+            classes: 'govuk-!-font-weight-bold'
+          },
+          attributes: {
+            'data-name': 'person[%index%][first_name]',
+            'data-id': 'person[%index%][first_name]'
+          }
+        }) }}
+
+        {{ govukInput({
+          id: 'person[0][last_name]',
+          name: 'person[0][last_name]',
+          label: {
+            html: 'Last name',
+            classes: 'govuk-!-font-weight-bold'
+          },
+          attributes: {
+            'data-name': 'person[%index%][last_name]',
+            'data-id': 'person[%index%][last_name]'
+          }
+        }) }}
+
+        {{ govukDateInput({
+          id: "date-of-birth",
+          namePrefix: "dob",
+          fieldset: {
+            legend: {
+              text: "Date of birth",
+              classes: "govuk-fieldset__legend--s"
+            }
+          },
+          hint: {
+            text: "For example, 31 3 1980"
+          },
+          items: [
+            {
+              id: "person[0][dob][day]",
+              name: "day",
+              classes: "govuk-input--width-2"
```

Comment:
You need to add attributes onto the date inputs like we do for the fields above to make the labels accessible when cloning. The component is fine. Just the demo that needs sorting.

Reply:
Done